std/experimental/color/package.d

```diff
@@ -0,0 +1,234 @@
+module std.experimental.color;
+
+public import std.experimental.color.rgb;
+import std.experimental.color.xyz: isXYZ, isxyY;
+
+@safe: pure: nothrow: @nogc:
+
+
+enum isColor(T) = isRGB!T || isXYZ!T || isxyY!T;
+
+enum isValidComponentType(T) = isIntegral!T || isFloatingPoint!T;
+
+
+// declare some common color types
+alias RGB8 =    RGB!("rgb", ubyte);
+alias RGBX8 =   RGB!("rgbx", ubyte);
+alias RGBA8 =   RGB!("rgba", ubyte);
+
+alias RGBf32 =  RGB!("rgb", float);
+alias RGBAf32 = RGB!("rgba", float);
+
+alias BGR8 =    RGB!("bgr", ubyte);
+alias BGRX8 =   RGB!("bgrx", ubyte);
+alias BGRA8 =   RGB!("bgra", ubyte);
+
+alias L8 =      RGB!("l", ubyte);
+alias A8 =      RGB!("a", ubyte);
+alias LA8 =     RGB!("la", ubyte);
+
+alias UV8 =     RGB!("rg", byte);
+alias UVf32 =   RGB!("rg", float);
+
+
+// colors from the WC3(/X11) color definitions
+enum Colors
```

Comment:
I see nothing wrong with saying the originating spec in documentation.
What I have a problem with is all lower case enum members.
Just errr.